### ISSUE TITLE
Outgoing SendGrid: Verified domain however SMTP 550 error prevents sending

### ISSUE THREAD

**Issue by jike212** [chatwoot/chatwoot#2371]
I'm running a new installation of Chatwoot 1.16.2 and I'm running into an issue with outgoing emails using SendGrid.

I've verified the wildcard sending domain in SendGrid and added the following environment variables"
```
SMTP_ADDRESS=smtp.sendgrid.net
SMTP_AUTHENTICATION=plain
SMTP_DOMAIN=<root domain from the domain authentication page on SendGrid>
SMTP_ENABLE_STARTTLS_AUTO=true
SMTP_PORT=587
SMTP_USERNAME=apikey
SMTP_PASSWORD=<my Sendgrid API key>
```

The error I get appears in the worker container logs and reflects the following:
```
WARN: Net::SMTPFatalError: 550 The from address does not match a verified Sender Identity. Mail cannot be sent until this error is resolved. Visit https://sendgrid.com/docs/for-developers/sending-email/sender-identity/ to see the Sender Identity requirements
```

Where is the `from` address set? I can't tell what it's sending to SendGrid

**Comment by sojan-official** [comment]
@jike212  Please configure `MAILER_SENDER_EMAIL`  environment variable

**Comment by jike212** [comment]
@sojan-official That did the trick, thank you!

**Comment by jike212** [comment]
This issue appears to have popped up again (I've now upgraded to 1.17.1) and it's not clear why. The domain is authenticated, I've even added a single-sender verification account with the same address as the inbox in Chatwoot but I still get the same SMTP 550 error.

**Comment by jike212** [comment]
Further update:

I rolled my instance back to a previous snapshot of the VM using 1.16.2 and then ran the update to 1.17.1 which got things working again. Not sure what might have broken in the previous upgrade attempt but I'll close the issue.

**Comment by latrocha** [comment]
Hi
I got the same issue. I solved it by using the same domain in the first admin user as the MAILER_SENDER_EMAIL enviroment variable.
For example, if you are using support.example.com as MAILER_SENDER_EMAIL, then create the first admin user as admin@support.example.com. Agents and admins created after that will work flawlessly but, if you use a different domain on the first user, it won't work.
It's weird, but that's how I solved it.
